src/app/components/search/search.component.html

```diff
@@ -1,38 +1,42 @@
-<div class='search'>
-  <mat-form-field appearance='fill' class='search__input'>
-    <mat-label i18n='Product search label@@product-search.label'>
+<div class="search">
```

Comment:
Please revert this file, single quotes are preferable.

Reply:
are you sure? 
Never heard that single quotes preferable in *.html files, and all Google examples that involve *.html files use double quotes in them.

Reply:
Yes, my bad, misplaced the comment.